01/Table.js

```diff
@@ -1,12 +1,36 @@
-import React from 'react';
+import React from "react";
+import TableHeader from "./components/TableHeader";
+import TableBody from "./components/TableBody";
+import TableFooter from "./components/TableFooter";
 
 class Table extends React.Component {
-    render() {
-        const {data} = this.props;
-        console.log(data);
-
-        return <table></table>
-    }
+  render() {
+    return (
+      <table>
+        <TableHeader headerNames={this.getHeader()} />
+        <TableBody bodyData={this.getBody()} />
+        <TableFooter data={this.getTotalPrice()}></TableFooter>
```

Comment:
TO co robiliśmy na spotkaniu byłoby ok, tylko wtedy do futer, przekazujemy np. `data={['', '', '', totalPrice]}` - mamy 4 kolumny, gdzie 3 pierwsze są puste.